oshdb-tool/etl/src/main/java/org/heigit/ohsome/oshdb/tool/importer/load/LoaderKeyTables.java

```diff
@@ -74,18 +75,16 @@ public void loadTags() {
     }
   }
 
-  public void loadRoles() {
+  public void loadRoles() throws FileNotFoundException, IOException {
```

Comment:
`FileNotFoundException` is redundant, isn't it?

Reply:
FileNotFoundExceptions got no removed where ever possible.